### DISCUSSION TITLE
Run GeneActivity Error.

### DISCUSSION THREAD

**Discussion by tzhu-bio** [stuart-lab/signac#680] [General]
Thanks for your signac!

When I Run GeneActivity, I got this error.
![image](https://user-images.githubusercontent.com/54537420/122663382-99df1680-d1cc-11eb-8ca3-1ecace37567c.png)

This is how I constructed the gene annotation using the gtf file.
![image](https://user-images.githubusercontent.com/54537420/122663427-ffcb9e00-d1cc-11eb-8fdc-b974baf6c54b.png)

Can you give me some advice?

**Comment by timoast** [comment]
Please open an issue including the complete code (as text not as screenshots) and output of `sessionInfo()`
